turbinia/workers/grep.py

```diff
@@ -36,11 +38,14 @@ def run(self, evidence, result):
     Returns:
         TurbiniaTaskResult object.
     """
-
-    patterns = evidence.config.get('filter_patterns')
-    if not patterns:
+    patterns_file = self.recipe['filter_patterns_file']
+    if patterns_file:
```

Comment:
I think you still want a `not` in here (whether it's the filter file or the filter list).